<ui version="4.0" >
 <class>WdgTransformationEffect</class>
 <widget class="QWidget" name="WdgTransformationEffect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Transformation Effects</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Mask Parameters</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Mask Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="maskName" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Scale X:</string>
        </property>
        <property name="buddy" >
         <cstring>label</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KDoubleNumInput" name="dblScaleX" >
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Scale Y:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KDoubleNumInput" name="dblScaleY" >
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Shear X:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="KDoubleNumInput" name="dblShearX" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Shear Y:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="KDoubleNumInput" name="dblShearY" />
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Rotation angle (radians):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="KDoubleNumInput" name="dblRotation" />
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Move X:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="KIntNumInput" name="intMoveX" />
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Move Y:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" >
       <widget class="KIntNumInput" name="intMoveY" />
      </item>
      <item row="8" column="0" >
       <widget class="QLabel" name="lblFilter" >
        <property name="text" >
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" >
       <widget class="KisCmbIDList" name="cmbFilter" />
      </item>
      <item row="9" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KisCmbIDList</class>
   <extends>QComboBox</extends>
   <header>widgets/kis_cmb_idlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
